SpaceDash/Controller/ViewControllers/RocketViewController.swift

```diff
@@ -0,0 +1,261 @@
+            
+            latestLaunchMetric.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            latestLaunchMetric.heightAnchor.constraint(equalToConstant: 60),
+            latestLaunchMetric.trailingAnchor.constraint(equalTo: rocketImage.leadingAnchor, constant: -40),
+            latestLaunchMetric.topAnchor.constraint(equalTo: upcomingLaunchMetric.bottomAnchor, constant: 25),
+            
+            firstLaunchMetric.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            firstLaunchMetric.heightAnchor.constraint(equalToConstant: 60),
+            firstLaunchMetric.trailingAnchor.constraint(equalTo: rocketImage.leadingAnchor, constant: -40),
+            firstLaunchMetric.topAnchor.constraint(equalTo: latestLaunchMetric.bottomAnchor, constant: 25),
+            
+            engineAndThrustMetric.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            engineAndThrustMetric.heightAnchor.constraint(equalToConstant: 60),
+            engineAndThrustMetric.trailingAnchor.constraint(equalTo: rocketImage.leadingAnchor, constant: -40),
+            engineAndThrustMetric.topAnchor.constraint(equalTo: firstLaunchMetric.bottomAnchor, constant: 25),
+            
+            descriptionMetric.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            descriptionMetric.heightAnchor.constraint(equalToConstant: 110),
+            descriptionMetric.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            descriptionMetric.topAnchor.constraint(equalTo: engineAndThrustMetric.bottomAnchor, constant: 30),
+            
+            readMoreBtn.topAnchor.constraint(equalTo: descriptionMetric.bottomAnchor, constant: -15),
+            readMoreBtn.heightAnchor.constraint(equalToConstant: 30),
+            readMoreBtn.widthAnchor.constraint(equalToConstant: 90),
+            readMoreBtn.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            bottomBlackView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            bottomBlackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            bottomBlackView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            bottomBlackView.topAnchor.constraint(equalTo: descriptionMetric.bottomAnchor, constant: -20),
+            
+            pathView.heightAnchor.constraint(equalToConstant: 70),
+            pathView.topAnchor.constraint(equalTo: bottomBlackView.topAnchor),
+            pathView.leadingAnchor.constraint(equalTo: bottomBlackView.leadingAnchor),
+            pathView.trailingAnchor.constraint(equalTo: bottomBlackView.trailingAnchor)
+        ])
+    }
+    
+    /// Custom navigation bar settings
+    private func setCustomNavBar(){
+        navigationItem.title = "Rockets"
+        navigationController?.navigationBar.isTranslucent = false
+        navigationController?.navigationBar.barTintColor = .white
```

Comment:
We are using a custom colour named DashCream for white. Use that colour instead of white everywhere.